web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -0,0 +1,123 @@
+  } else if (estimationData.isAggregated) {
+    if (estimationData.isEstimated) {
+      return 'The data consists of summarised values of hourly recordings throughout the day. Some hourly data is estimated.';
+    }
+    return 'The data consists of summarised values of hourly recordings throughout the day.';
+  }
+  return 'The data for this hour has not yet been reported. The displayed values are estimates and will be replaced with measured data once available.';
+}
+
+export default function EstimationCard({
+  estimationData,
+}: {
+  estimationData: EstimationProperties;
+}) {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const handleToggleCollapse = () => {
+    setIsCollapsed((previous) => !previous);
+  };
+  const bgColorClasses = estimationData.isOutage
+    ? 'bg-amber-700/20 dark:bg-amber-500/20'
+    : 'bg-neutral-100 dark:bg-gray-800';
+  const textColorTitle = estimationData.isOutage
+    ? 'text-amber-700 dark:text-amber-500'
+    : estimationData.isAggregated
+    ? 'text-black dark:text-white'
+    : 'text-amber-700 dark:text-amber-500';
+  const textColorBody = 'text-neutral-600 dark:text-neutral-400';
+  const textColorLink = 'text-black dark:text-white';
+  const borderColor = 'border-neutral-200 dark:border-gray-700';
+  const icon = estimationData.isOutage
+    ? "bg-[url('/images/estimated_light.svg')] dark:bg-[url('/images/estimated_dark.svg')]"
+    : estimationData.isAggregated
+    ? "bg-[url('/images/aggregated_light.svg')] dark:bg-[url('/images/aggregated_dark.svg')]"
+    : "bg-[url('/images/estimated_light.svg')] dark:bg-[url('/images/estimated_dark.svg')]";
+
+  const pillType = estimationData.isOutage
+    ? 'outage'
+    : estimationData.isAggregated
+    ? 'aggregated'
+    : 'estimated';
+
+  return (
+    <div
+      className={`w-full rounded-lg px-3 py-2.5 ${
+        isCollapsed ? 'h-[46px]' : 'h-fit'
+      } ${bgColorClasses} border ${borderColor} mb-4 gap-2 transition-all`}
+    >
+      <div className="flex flex-col">
+        <button onClick={handleToggleCollapse}>
+          <div className="flex flex-row justify-between pb-1.5">
+            <div className="flex flex-row gap-2">
+              <div className={`flex items-center justify-center`}>
+                <div className={`h-[16px] w-[16px] bg-center ${icon}`} />
+              </div>
+              <h2
+                className={`truncate text-sm font-semibold ${textColorTitle} self-center`}
+              >
+                {GetTitle(estimationData)}
+              </h2>
+            </div>
+            <div className="flex flex-row gap-2 ">
+              {(estimationData.isEstimated || estimationData.isOutage) && (
+                <Badge type={pillType}>{GetPillText(estimationData)}</Badge>
+              )}
+              <div className="text-lg">
+                {isCollapsed ? <HiChevronUp /> : <HiChevronDown />}
```

Comment:
Are these not inverted?
<img width="439" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/0d06b64c-d788-488a-89a6-65e6ad0b5fca">
<img width="439" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/9856bae5-9587-465b-9f55-2c08177a9de0">

I think I would prefer them the other way around.

Reply:
I'll ask @Alportan about it

Reply:
We couldn't figure out what's the right way here, but [Material UI](https://mui.com/material-ui/react-accordion/) and [shadcn/ui](https://ui.shadcn.com/docs/components/accordion) does as you suggest, so let's go with that 😄

Reply:
Ohh good catch Viktor, I'll update Figma to match reality. 🙌